src/main/java/io/vertx/ext/hawkular/impl/inventory/EntityReporter.java

```diff
@@ -95,7 +95,7 @@ protected static void createFeed(Future<Void> fut) {
 
   protected static void createResourceType(JsonObject body, Future<Void> fut) {
     HttpClientRequest request = httpClient.post(composeEntityUri("f;"+feedId, "resourceType"), response -> {
-      if (response.statusCode() == 201) {
+      if (response.statusCode() == 201 || response.statusCode() == 409) {
```

Comment:
Yes, some resources share the same resource types so 409 means the resource type is already created.